lib/commit-analyzer.js

```diff
@@ -0,0 +1,33 @@
+const { exec } = require('child_process');
+
+const cmd = (cmd, cb) => exec(cmd, { cwd: __dirname }, (err, stdout) => cb(stdout.split('\n').join('')));
```

Comment:
Can you move `cmd` into some sort of util and use it here and in bin/github-publish-release